### PR TITLE
Add orjson dependency in setup.py

### PR DESCRIPTION
**Pull request by Jasopaum**
I had a problem in fhirstore because this dependency was not installed

### CODE REVIEW

File: src/fhirpath/search.py [hunk 1]
```diff
@@ -354,7 +354,7 @@ def build_query_params_string(self, params):
         # Otherwise, an expection is raised for None values
         params = {k: v for k, v in params.items() if v is not None}
         return URL.build(
-            query={**params, **{"_type": self.context.resource_types}}
+            query={**params, **{"_type": ",".join(self.context.resource_types)}}
```
Comment (simonvadee) [inline src/fhirpath/search.py:357]: did you mean to commit that in this PR ? doesn't it fix the arkhn/warehouse-api#169 issue ?

Comment (Jasopaum) [reply src/fhirpath/search.py:357]: Hum, I think it was related to the solving of this issue, yeah! I thought it was something I had forgotten from another PR
Do you think I should split it in 2 PRs (or at least 2 commits)?

Comment (simonvadee) [reply src/fhirpath/search.py:357]: two commits and then rebase sounds good :)

Comment (Jasopaum) [reply src/fhirpath/search.py:357]: Done 🙂

**Review by simonvadee — APPROVED** [review]

**Pull request merged**